transformations/multilingual_dictionary_based_code_switch/README.md

```diff
@@ -0,0 +1,84 @@
+# Multi-lingual code-switch 🦎  + 📕 -> 🐍
+
+Author name: Libo Qin, Fuxuan Wei, Tianbao Xie, Wanxiang Che, Yue Zhang
+
+Author email: {lbqin, fuxuanwei, tianbaoxie, car}@ir.hit.edu.cn, yue.zhang@wias.org.cn
+
+Author Affiliation: Harbin Institute of Technology
+
+## What type of a transformation is this?
+This transformation can be seen as a data augmentation framework to generate multi-lingual code-switching data to fine-tune mBERT, which encourages model to align representations from source and multiple target languages once by mixing their context information.
+
+## What tasks does it intend to benefit?
+This alignment of different language it brought can benefit a massive of multi-lingual tasks.
+As mentioned in the paper, we test this method on tasks including these multi-lingual tasks: 
+Natural Language Inference, Sentiment Classification, Document Classification, Dialogue State Tracking (DST), Spoken Language Understanding
+We perform t-test for all experiments to measure whether the results from the proposed model are significantly better than the baselines(mBERT and XLM).
+
+## What are the limitations of this transformation?
+It is worth noticing that words in the source language can have multiple translations in the target language. In this case, we randomly choose any of the multiple translations as the replacement target language word. Though we cannot guarantee that this is the correct word-to-word translation in the context, we can consider it as one of the data augmented strategy for our tasks. This is verified by our work (CoSDA-ML) [1].
```

Comment:
We also mention this in our code-mixing paper for our PolyGloss attack and propose a method to disambiguate translations, but that requires a sentence translation.

Reply:
@samsontmr Sure, we have added your good work on the section of What are the limitations of this transformation? Thanks again!